src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -756,6 +775,23 @@ export default {
           comments: this.flattenAndSortComments(header.comments)
         }
 
+        // Apply additional properties for Staff Only Filings
+        if (this.isStaffFiling(filingType)) {
+          let baseFiling
+          switch (filingType) {
+            case FilingTypes.REGISTRARS_NOTATION: baseFiling = filing.registrarsNotation
+              break
+            case FilingTypes.REGISTRARS_ORDER: baseFiling = filing.registrarsOrder
+              break
+            case FilingTypes.COURT_ORDER: baseFiling = filing.courtOrder
+              break
+          }
+
+          item.notationOrOrder = baseFiling.orderDetails
```

Comment:
If this code ever gets out of sync with `isStaffFiling()` then it's possible none of these cases apply and baseFiling will remain undefined, causing an error here.
You could init baseFiling to {}, or use conditional chaining.

Reply:
ah interesting! good catch. I will add conditional chaining 👍